File: ueueo-framework/ueueo-object-extending/src/main/java/com/ueueo/data/objectextending/modularity/ExtensionPropertyLookupConfiguration.cs
using Microsoft.Extensions.Logging;

namespace Volo.Abp.ObjectExtending.Modularity;

public class ExtensionPropertyLookupConfiguration
{
    public String Url;// { get; set; }

    /**
     * Default value: "items".
    */
    public String ResultListPropertyName;// { get; set; } = "items";

    /**
     * Default value: "text".
    */
    public String DisplayPropertyName;// { get; set; } = "text";

    /**
     * Default value: "id".
    */
    public String ValuePropertyName;// { get; set; } = "id";

    /**
     * Default value: "filter".
    */
    public String FilterParamName;// { get; set; } = "filter";
}

File: abp-frameworks/ueueo-object-extending/src/main/java/com/ueueo/data/objectextending/modularity/ExtensionPropertyLookupConfiguration.java
package com.ueueo.data.objectextending.modularity;

import lombok.Data;

/**
 * @author Lee
 * @date 2022-05-23 14:25
 */
@Data
public class ExtensionPropertyLookupConfiguration {
    private String url;

    /** Default value: "items". */
    public String ResultListPropertyName = "items";

    /** Default value: "text". */
    public String DisplayPropertyName = "text";

    /** Default value: "id". */
    public String ValuePropertyName = "id";

    /** Default value: "filter". */
    public String FilterParamName = "filter";
}
